deploy.Jenkinsfile

```diff
@@ -0,0 +1,97 @@
+def deployNode (host) {
+    return {
+         sshagent (credentials: ['buildagent-matcher']) {
+             sh "ssh -o StrictHostKeyChecking=no -l buildagent-matcher ${host} hostname"
+
+             sh "scp ./waves/node/target/waves-devnet*all*.deb buildagent-matcher@${host}:/home/buildagent-matcher"
+             sh "scp ./matcher/dex-load/src/main/resources/reinstallNode.sh buildagent-matcher@${host}:/home/buildagent-matcher"
+             sleep time: 80000, unit: 'MILLISECONDS'
+             sh "ssh -q buildagent-matcher@${host} sudo sh reinstallNode.sh"
+         }
+    }
+}
+
+pipeline {
+    agent {
+        label 'buildagent-matcher'
+    }
+    environment {
+        SBT_HOME = tool name: 'sbt-1.2.6', type: 'org.jvnet.hudson.plugins.SbtPluginBuilder$SbtInstallation'
+        SBT_THREAD_NUMBER = "6"
+        SBT_OPTS = '-Xmx2g -XX:ReservedCodeCacheSize=128m -XX:+CMSClassUnloadingEnabled -Dnetwork=devnet'
+        PATH = "${env.SBT_HOME}/bin:${env.PATH}"
+    }
+    stages {
+        stage('Checkout') {
+            steps {
+                cleanWs()
+                dir('matcher') {
+                    checkout scm: [$class: 'GitSCM', userRemoteConfigs: [[url: 'https://github.com/wavesplatform/matcher.git', credentialsId: null]], extensions: [[$class: 'CloneOption', noTags: false, reference: '', shallow: false]], branches: [[name: '*/DEX-822']]], poll: false
+                }
+                dir('waves') {
+                    checkout scm: [$class: 'GitSCM', userRemoteConfigs: [[url: 'https://github.com/wavesplatform/waves.git', credentialsId: null]], extensions: [[$class: 'CloneOption', noTags: false, reference: '', shallow: false]], branches: [[name: '*/master']]], poll: false
+
+                }
+            }
+        }
+        stage('Compile Matcher') {
+            steps {
+                dir('matcher') {
+                    sh 'printenv'
+                    sh 'git fetch --tags'
+                    sh 'find ~/.sbt/1.0/staging/*/waves -type d -name target | xargs -I{} rm -rf {}'
+                    sh 'find . -type d -name target | xargs -I{} rm -rf {}'
+                    sh 'sbt "set Global / scalacOptions ++= Seq(\\"-Xfatal-warnings\\", \\"-Ywarn-unused:-imports\\");session save;cleanAll;compile;packageAll"'
```

Comment:
We don't need here warn checks, so just:
```
sh 'sbt packageAll'
```

Reply:
ok